report/3.Theory/SemanticsDescription.tex

```diff
@@ -464,15 +464,26 @@ \subsection{Big-Step Semantic for Expressions}
 The transition-system for $tuple_{exp}$ is $(tuple_{tuple_exp} \cup Tuple, \rightarrow_{tuple}, Tuple)$.
 The transitions are shown below.
 
-\texttt{[$TplAt_{BSS}$]}\\
+\texttt{[$TupleElementAtIndexHit_{BSS}$]}
+\begin{center}
+	\begin{math}
+	\cfrac
+	{env_V, sto \vdash e \rightarrow_{e} v' \quad env_V, sto \vdash tpl_v' \rightarrow_e v}
+	{env_V, sto \vdash tuple\ element e \rightarrow_{tuple} v}
```

Comment:
Jeg synes ikke, at dette er rigtigt, fordi det er ikke et expression, der sættes efter "element". Elementerne hedder rent faktisk element0/element1/etc.
Der kan skrives "element i" hvor 0 <= i <= tuple-størrelse - 1

Reply:
Ja fuldstændig enig. Det er fikset til ikke at være et udtryk men blot et heltal. Jeg dog ladet sidebetingelserne stået som de var med de nye noteringer, da jeg syntes at de giver det samme information. Plus at de forklare hvad der faktisk sker. Så slipper vi for at have endnu en linje.